### DISCUSSION TITLE
Is `material-forum-editor-buttons`'s clean icon really the best option?

### DISCUSSION THREAD

**Discussion by just-a-hriday** [ScratchAddons/ScratchAddons#4242] [Discussion]
Right now, `material-forum-editor-buttons` uses <sub>![format_clear](https://fonts.gstatic.com/s/i/materialiconsoutlined/format_clear/v12/24px.svg)</sub> for the second-last button which cleans up your BBCode.
Honestly though, I don't think it's a very good fit. Maybe <sub>![cleaning_services](https://fonts.gstatic.com/s/i/materialiconsoutlined/cleaning_services/v12/24px.svg)</sub> would suit it better?

**Comment by CST1229** [comment]
Well, the current icon does somewhat indicate what the button does (remove BBCode).

**Reply by just-a-hriday** [reply]
Meh. It's more like a button to clear _all_ formatting.

**Reply by mxmou** [reply]
The T icon means exactly that, at least in Google Docs.
![image](https://user-images.githubusercontent.com/51849865/153145100-cd1face0-f809-46bc-be8b-c6584209f768.png)

**Reply by just-a-hriday** [reply]
> The T icon means exactly that, at least in Google Docs.

In Docs, that button _clears_ all formatting. So it just straight up deletes all the formatting you've done.

**Reply by Secret-chest** [reply]
> > The T icon means exactly that, at least in Google Docs.
> 
> In Docs, that button _clears_ all formatting. So it just straight up deletes all the formatting you've done.

And dosen't removing BBCode do the same thing?

**Reply by just-a-hriday** [reply]
> > > The T icon means exactly that, at least in Google Docs.
> > 
> > 
> > In Docs, that button _clears_ all formatting. So it just straight up deletes all the formatting you've done.
> 
> And dosen't removing BBCode do the same thing?

Well, that button in the forums doesn't seem to remove _any_ BBCode, honestly.

**Reply by CST1229** [reply]
> Well, that button in the forums doesn't seem to remove _any_ BBCode, honestly.

It deletes the BBCode of the currently selected text.
